### PR TITLE
added event create api endpoint

### PR DESCRIPTION
**Pull request by Skovy**

### CODE REVIEW

**Comment by Skovy** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;2t61kU9K&#x2F;1038-events-api-docs"> Events API docs</a>

**Comment by rjnordeen** [comment]
+1

File: api.rst [hunk 1]
```diff
@@ -619,3 +619,43 @@ page                  The page of results that you want, 0-based.
...
+**Params**
+
+=============================== ==================================================================
+Key                             Value
+=============================== ==================================================================
+\*start_date                    (String) The date and time when the event starts. Ex: '2017-03-10 9:00 AM'
+\*end_date                      (String) The date and time when the event ends. Ex: '2017-03-10 10:00 AM'
+\*name                          (String) The name of the event.
+\*event_type_name               (String) The type of event. Possible types: 'Workshop', 'Info Session', 'Group Appointment', 'Other'
+status                          (String) The status of the event. Possible status: 'approved', 'pending', 'declined'
```
Comment (jordondornbos) [inline api.rst]: @Skovy Might be nice to note that it will default to 'pending'

File: api.rst [hunk 2]
```diff
@@ -619,3 +619,43 @@ page                  The page of results that you want, 0-based.
...
+description                     (String) The description of the event.
+student_registration_start      (String) The date students can start registering. Ex: '2017-03-01 7:00 AM'
+student_registration_end        (String) The date students can no longer register. Ex: '2017-03-09 8:00 PM'
+invite_only                     (Boolean) If the event requires an invite.
+attendee_limit                  (Integer) The number of attendees allowed to register.
```
Comment (jordondornbos) [inline api.rst]: @Skovy Might be nice to also allow sending `student_cost`

File: api.rst (hunk 1 above)
Comment (Skovy) [inline api.rst]: Good point. Will add.

File: api.rst (hunk 2 above)
Comment (Skovy) [inline api.rst]: @jordondornbos Are you sure that's currently an allowed parameter for the Events API?

Comment (jordondornbos) [inline api.rst]: Leaving as-is for now.

**Comment by jordondornbos** [comment]
@Skovy Looks good

**Pull request merged**